Cartfile

```diff
@@ -1,7 +1,7 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 4.3
 binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 5.0.0
-github "mapbox/MapboxDirections.swift" ~> 0.26.0
+github "mapbox/MapboxDirections.swift" "master"
```

Comment:
Ideally we’ll release MapboxDirections.swift v0.26.2 shortly before landing this PR.